recipes/soplex/all/conanfile.py

```diff
@@ -0,0 +1,124 @@
+from conan import ConanFile
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import collect_libs, copy, get, replace_in_file
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.scm import Version
+from os.path import join
+
+required_conan_version = ">=1.53.0"
+
+
+class SoPlexConan(ConanFile):
+    name = "soplex"
+    description = "SoPlex linear programming solver"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://soplex.zib.de"
+    topics = ("simplex", "solver", "linear", "programming")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "BOOST": [True, False],
+        "GMP": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "BOOST": False,
+        "GMP": False
```

Comment:
IS there a reason to not match the default https://github.com/scipopt/soplex/blob/4ab99fc9f52fcbcc738a90e59cbba1862eb71d92/CMakeLists.txt#L32?

Reply:
I changed the defaults to True now